src/test/miner_tests.cpp

```diff
@@ -250,13 +251,14 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
 //                BOOST_CHECK(nodePtr && nodePtr->IsActive(tip->height));
 //
 //                mintedBlocks = nodePtr->mintedBlocks;
+//                creationHeight = int64_t(nodePtr->creationHeight);
 //            }
```

Comment:
Can remove all these commented code

Reply:
Probably we can enabled it in the future?